doc/btp_spec.txt

```diff
@@ -1372,7 +1372,9 @@ Commands and responses:
 		Command parameters:	Address_Type (1 octet)
 					Address (6 octets)
 					PSM (2 octets)
-		Response parameters:	Chan_ID (1 octet)
+					MTU (2 octets)
+					Num (1 octets)
+		Response parameters:	Chan_ID (1 octet x num of channels created)
```

Comment:
It would be good to add Num as first parameter to have the same convention as for other commands